source/openapi-admin-v3.yaml

```diff
@@ -7191,10 +7191,31 @@ components:
             Protocol](https://www.mongodb.com/docs/atlas/app-services/mongodb/wire-protocol/#connect-over-the-wire-protocol).
     FlexibleSync:
       type: object
+      required: 
+        - state
+        - development_mode_enabled
       properties:
         state:
           type: string
-          description: The state of the sync service ("enabled", "disabled", or empty if Sync is not configured).
+          description: The state of the sync service (`"enabled"`, `"disabled"`, or empty if Sync is not configured).
+        development_mode_enabled:
```

Comment:
I need to confirm, but I think interestingly developer_mode_enabled and service_name are not included in this payload

Reply:
Would be curious if you dont mind setting up an app with all of the options and hitting the endpoint. My understanding and reading of the code are that the two fields are not returned here (for some odd reason we decided that developer mode was an app setting and not a service setting so its define in the app/sync handler). That is a different endpoint and I think interestingly one that is not documented on here. 

TLDR, if you hit this endpoint: `"/groups/%s/apps/%s/sync/config"` you will get the following data: 

```
type syncData struct {
	DevelopmentModeEnabled bool   `json:"development_mode_enabled"`
	ServiceID              string `json:"service_id,omitempty"`
}
```

Reply:
Confirmed this is correct. Reading the ticket I am curious if these fields were seen in the API? The weird thing (which I totally get) is that these fields are a part of the sync.json field in the cli download, so I am wondering if that is what was being confused cc @lindseymoore  / @nlarew

Reply:
Thanks for checking and for pointing out the difference between the service config and the app's sync config. I can remove these two options from this API reference since they are not returned @tkaye407